.github/workflows/build.yml

```
@@ -170,6 +170,32 @@ jobs:
         if: startsWith(matrix.os, 'ubuntu') && startsWith(matrix.arch, 'x86_64') && startsWith(matrix.compiler, 'clang')
         run: |
           sudo apt-get install ${{ matrix.compiler }}
+      - name: Setup Windows prerequisites
+        if: startsWith(matrix.os, 'windows')
+        uses: msys2/setup-msys2@v2
+        with:
+          msystem: MINGW64
+          update: true
+          install: git mingw-w64-x86_64-toolchain automake1.16 automake-wrapper autoconf libtool make mingw-w64-x86_64-json-c mingw-w64-x86_64-crt-git mingw-w64-x86_64-pcre
+      - name: Setup Windows prerequisites (libgcrypt)
+        if: startsWith(matrix.os, 'windows') && startsWith(matrix.gcrypt, '--with-local-libgcrypt')
+        run: |
+          msys2 -c 'git clone --branch libgpg-error-1.42 https://github.com/gpg/libgpg-error'
+          cd libgpg-error
+          msys2 -c './autogen.sh'
```

Is it possible to merge it with the `./configure` step e.g.:
```suggestion
          msys2 -c './autogen.sh --enable-maintainer-mode --enable-static --enable-shared --with-pic --disable-doc --disable-nls'
```
So we can safe one additional `./configure` step.

---

Sorry, this isn't libndpi...